install/kubernetes/helm/istio/charts/tracing/templates/service-jaeger.yaml

```diff
@@ -62,6 +62,26 @@ items:
     selector:
       app: jaeger
     type: ClusterIP
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: jaeger-collector-headless
+    namespace: {{ .Release.Namespace }}
+    labels:
+      app: jaeger
+      jaeger-infra: collector-service
+      chart: {{ template "tracing.chart" . }}
+      heritage: {{ .Release.Service }}
+      release: {{ .Release.Name }}
+  spec:
+    ports:
+    - name: jaeger-collector-grpc
+      port: 14250
+      targetPort: 14250
+      protocol: TCP
```

Comment:
We already has this port exposed in service `jaeger-collector`

Reply:
The `jaeger-collector` service isn't headless. For gRPC client-side load balancing to work properly, the service has to be headless.